Makefile

```diff
@@ -196,7 +196,7 @@ test: all
 	$(MAKE) build-addons-napi
 	$(MAKE) cctest
 	$(PYTHON) tools/test.py --mode=release -J \
-		addons addons-napi doctool inspector known_issues message pseudo-tty parallel sequential
+		addons addons-napi doctool inspector known_issues message pseudo-tty parallel sequential async-hooks
```

Comment:
Could you break at 80 characters? Looks like two of the extra lines here got longer when rebasing on top of the `addons-napi` addition